.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We usually don't include editor settings in repositories, since the `.editorconfig` file already implies some things. For VS Code you can use the [EditorConfig extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)